desktop/release-notes.md

```diff
@@ -188,7 +188,7 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/ge
 - Fixed issues related to performing actions on selected containers. [Fixes https://github.com/docker/for-win/issues/13005](https://github.com/docker/for-win/issues/13005)
 - Added functionality that allows you to display timestamps for your container or project view.
 - Fixed a possible segfault when interrupting `docker pull` with Control+C.
-- Increased the default DHCP lease time to avoid the VM's network glitching and dropping connections every two hours.
+- Increased the default DHCP lease time to avoid the VM's ne2rk glitching and dropping connections every 2 hours.
```

Comment:
😅 looks like a "find and replace" whoopsie 😂 

```suggestion
- Increased the default DHCP lease time to avoid the VM's network glitching and dropping connections every two hours.
```